Add Standard Library content to the Swan Lake Beta 3 release note (#922)
ballerina-platform/ballerina-release: release-notes/swan-lake-beta3-release-note.md

#### New Features

##### Crypto Package

> - Improve hash APIs for cryptographic salt.

Reviewer: ```suggestion
- Improved the hash APIs for cryptographic salt
```

##### GraphQL Package

> - Add field alias support for GraphQL documents.

Reviewer: ```suggestion
- Added field alias support for GraphQL documents
```

- Add variable support in GraphQL requests.

> - Add mutation support for GraphQL services.

Reviewer: ```suggestion
- Added mutation support for GraphQL services
```

- Add typename introspection.

##### gRPC Package
- Add declarative auth configurations.
- Add Timestamp, Duration, and Struct type support.

##### HTTP Package

> - Enable HTTP trace and access log support.

Reviewer: ```suggestion
- Enabled HTTP trace and access log support
```

- Add HATEOAS link support.

> - Introduce `http:CacheConfig` annotation to the resource signature.

Reviewer: ```suggestion
- Introduced the `http:CacheConfig` annotation to the resource signature
```

> ##### GraphQL Package
> - Add field alias support for GraphQL documents.
> - Add variable support in GraphQL requests.
> - Add mutation support for GraphQL services.
> - Add typename introspection.
> 
> ##### HTTP Package
> - Enable HTTP trace and access log support.
> - Add HATEOAS link support.
> - Introduce `http:CacheConfig` annotation to the resource signature.
> - Introduce support for service specific media-type subtype prefix.
> - Introduce introspection resource method to get generated OpenAPI document of the service.
> 
> ##### JWT Package
> - Add HMAC signature support for JWT.
> 
> ##### gRPC Package
> - Add declarative auth configurations.
> - Add Timestamp, Duration, and Struct type support.
> 

Reviewer 2: ```suggestion
##### GraphQL Package
- Add field alias support for GraphQL documents.
- Add variable support in GraphQL requests.
- Add mutation support for GraphQL services.
- Add typename introspection.

##### gRPC Package
- Add declarative auth configurations.
- Add Timestamp, Duration, and Struct type support.

##### HTTP Package
- Enable HTTP trace and access log support.
- Add HATEOAS link support.
- Introduce `http:CacheConfig` annotation to the resource signature.
- Introduce support for service specific media-type subtype prefix.
- Introduce introspection resource method to get generated OpenAPI document of the service.

##### JWT Package
- Add HMAC signature support for JWT.

```

#### Improvements

##### GraphQL Package

> - Validate `maxQueryDepth` at runtime, as opposed to validating it on compile time.

Reviewer: ```suggestion
- Validate the `maxQueryDepth` at runtime as opposed to validating it at compile time
```

##### HTTP Package

> - Add support for `map<json>` as query parameter type.

Reviewer: ```suggestion
- Added support for the `map<json>` as query parameter type
```

> - Support nilable client data binding types.

Reviewer: ```suggestion
- Added support for nilable client data binding types
```

##### WebSocket Package

> - Make the WebSocket caller isolated.

Reviewer: ```suggestion
- Made the WebSocket caller isolated
```

> - Introduce write timeout for WebSocket client.

Reviewer: ```suggestion
- Introduced a write timeout for the WebSocket client
```
